### ISSUE TITLE
Add Tutorial

### ISSUE THREAD

**Issue by doppelganger23** [KnowledgeExpert/selenidejs#51]
It will be good to have a tutorial with simple test examples, like this one:
https://www.protractortest.org/#/tutorial

**Comment by AleksanderPopov** [comment]
@doppelganger23 hey, please take a look at draft tutorial

**Comment by doppelganger23** [comment]
I was trying to run tests doing steps as described in tutorial and runned into this error:

```
C:\work\try-selenide-js>npm test

> @ pretest C:\work\try-selenide-js
> tsc


> @ test C:\work\try-selenide-js
> jasmine ./build/*spec.js

Randomized with seed 22584
Started

<--- Last few GCs --->

[13220:0000026E40FBE4F0]    29046 ms: Mark-sweep 1430.5 (1494.5) -> 1430.4 (1463.0) MB, 956.6 / 0.0 ms  (+ 0.0 ms in 0 steps since start of marking, biggest step 0.0 ms, walltime since start of marking 1050 ms) last resort GC in
old space requested
[13220:0000026E40FBE4F0]    30055 ms: Mark-sweep 1430.4 (1463.0) -> 1430.4 (1463.0) MB, 1009.2 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0000033C910A5879 <JSObject>
    2: /* anonymous */(aka /* anonymous */) [C:\work\try-selenide-js\node_modules\selenidejs\built\actions.js:~453] [pc=0000027417F22D47](this=00000014A8F822D1 <undefined>,error=0000026E55DFA641 <TypeError map = 00000008F87E8B69>
)
    3: /* anonymous */(aka /* anonymous */)(this=00000014A8F822D1 <undefined>)
    7: runMicrotasksCallback(aka runMicrotasksCallback) [internal/process/next_tick.js...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
 1: node_module_register
 2: v8::internal::FatalProcessOutOfMemory
 3: v8::internal::FatalProcessOutOfMemory
 4: v8::internal::Factory::NewMap
 5: v8::internal::JSReceiver::GetOwnPropertyDescriptor
 6: v8::internal::JSReceiver::GetOwnPropertyDescriptor
 7: v8::internal::JSReceiver::GetOwnPropertyDescriptor
 8: v8::internal::JSReceiver::class_name
 9: v8::internal::StringHasher::ComputeUtf8Hash
10: v8::internal::StringHasher::ComputeUtf8Hash
11: v8::internal::StringHasher::ComputeUtf8Hash
12: v8::internal::StringHasher::ComputeUtf8Hash
13: v8::internal::Factory::NewJSGeneratorObject
14: v8::internal::compiler::RawMachineAssembler::MakeNode
15: 0000027417D843C1
npm ERR! Test failed.  See above for more details.
```

Could reproduce it with this minimal spec:
```
import { Browser } from 'selenidejs';
import './base'

describe('The Internet', () => {
    it('should have a title', async () => {
        await Browser.open('http://the-internet.herokuapp.com/');
    });
});
```
Base module is identical to yours from tutorial.
OS windows 10 enterprise.
Nodejs version 8.11.3.
Java version 1.8.0_162.

**Comment by AleksanderPopov** [comment]
Fixed #54
